pandas/core/series.py

```diff
@@ -3847,9 +3847,6 @@ def f(x):
             f = func
 
         with np.errstate(all="ignore"):
-            if isinstance(f, np.ufunc):
```

Comment:
why would we want to do this?
is going to kill perf

Reply:
Issue #33492 describes that. I also added test

Reply:
as I said I am not inclided to accept this w/o proof that this does not kill perf. We likely don't have asv's for this (or even using a np.ufunc in apply).

Now if you add those and either perf is not degraded, or you actually catch the TypeError then I would be ok.